assembly/std/arguments.ts

```diff
@@ -0,0 +1,165 @@
+import {Address} from './address';
+import {ByteArray} from '@massalabs/as/assembly/byteArray';
+
+/**
+ * Args for remote function call
+ */
+export class Args {
+  private offset: i64 = 0;
+  private argsString: string = '';
+
+  /**
+   *
+   * @param {string} argsString
+   * @param {Array<string>} types
+   */
+  constructor(argsString: string = '') {
+    this.argsString = argsString;
+  }
+
+  /**
+   *
```

Comment:
You should add documentation to all function explaining what it does.

Reply:
done